oojs.js

```diff
@@ -1,12 +1,33 @@
 // Define a Playlist class. It should have...
+//
   // A title property that is determined by some input.
   // A songs property that is determined by some input. It should contain multiple songs, each of which can just be a song title.
   // A favorites property that is initialized at 0. This cannot be set by user input.
   // An addSong method that adds a song to the songs property.
+  music class Playlist {
+    constructor(songTitle, genre){
+    this.songTitle = songTitle;
+    this.genre = genre;
+    this.songs = ["Achey Breaky Heart", "Bad to the Bone", "Miami"]
+    favorites = 0
```

Comment:
In order for `favorites` to be attached to the class and any instances generated from it, then it needs to be written as `this.favorites = 0`